bindings/java/mongocrypt/src/main/java/com/mongodb/crypt/capi/CAPI.java

```diff
@@ -545,6 +599,44 @@ public interface mongocrypt_random_fn extends Callback {
     public static native void
     mongocrypt_destroy(mongocrypt_t crypt);
 
+    /**
+     * Obtain a nul-terminated version string of the loaded csfle dynamic library,
+     * if available.
+     *
+     * If no csfle was successfully loaded, this function returns NULL.
+     *
+     * @param crypt The mongocrypt_t object after a successful call to mongocrypt_init.
+     * @param len an optional length of the returned string. May be NULL.
+     *
+     * @return A nul-terminated string of the dynamically loaded csfle library.
+     * @since 1.5
+     */
+    public static native cstring
+    mongocrypt_csfle_version_string(mongocrypt_t crypt, Pointer len);
+
+    /**
+     * Obtain a 64-bit constant encoding the version of the loaded csfle
+     * library, if available.
+     *
+     * <p>The version is encoded as four 16-bit numbers, from high to low:
+     * <ol>
+     *   <li>Major version</li>
+     *   <li>Minor version</li>
+     *   <li>Revision</li>
+     *   <li>Reserved</li>
+     * </ol>
+     * For example, version {@code 6.2.1} would be encoded as: {@code 0x0006'0002'0001'0000}
+     * </p>
+     *
+     * @param  crypt The mongocrypt_t object after a successful call to
+     * mongocrypt_init.
+     *
+     * @return A 64-bit encoded version number, with the version encoded as four
+     * sixteen-bit integers, or zero if no csfle library was loaded.
+     * @since 1.5
+     */
+    public static native long
+    mongocrypt_csfle_version(mongocrypt_t crypt);
```

Comment:
It doesn't seem like this is used.  Should it be?  If not, does it need to be defined?